src/logging.cpp

```diff
@@ -174,7 +177,7 @@ std::vector<CLogCategoryActive> ListActiveLogCategories()
     return ret;
 }
 
-std::string BCLog::Logger::LogTimestampStr(const std::string &str)
+std::string BCLog::Logger::LogTimestampStr(const std::string str)
```

Comment:
In commit "threads: prefix log messages with thread names" (4fba3ead9811ebe47161e9e7f535a7cc88403cf5)

Not sure if you dropped the `&` intentionally here. It would make sense to drop it, so the string could be moved from instead of copied. But you'd also want to drop the `const` and add `std::move` below, and also change the `Logger::LogPrintStr` function similarly (function calling this function).

Without these things, I don't think there'd be a reason to drop `&`.